dashboard/app/models/user_permission.rb

```diff
@@ -41,7 +41,9 @@ class UserPermission < ActiveRecord::Base
     # professional development workshop attendance.
     WORKSHOP_ORGANIZER = 'workshop_organizer'.freeze,
     # Grants ability to conduct peer reviews for professional learning courses
-    PLC_REVIEWER = 'plc_reviewer'.freeze
+    PLC_REVIEWER = 'plc_reviewer'.freeze,
+    # Makes the account satisfy authorized_teacher?
```

Comment:
Something like "An Authorized Teacher account belongs to someone who is verified as a teacher. They can see answer keys, teacher-facing markdown, and other locked-down teacher-only content" would work